.github/PULL_REQUEST/new_config.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New Config
+about: A new cluster config
+---
+
+Please follow these steps before submitting your PR:
+
+* [ ] If your PR is a work in progress, include `[WIP]` in its title
+* [ ] Its commits' summaries are reasonably descriptive
+* [ ] Your PR targets the `master` branch
+* [ ] You've included links to relevant issues, if any
+
+* [ ] Add your custom config file to the `conf/` directory
+* [ ] Add your documentation file to the `docs/` directory
+* [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
+* [ ] Add your custom profile to the `README.md` file in the top-level directory
+* [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`
+
+Please see [uploading to
+`nf-core/configs`](https://github.com/nf-core/configs#uploading-to-nf-coreconfigs)
+for more details.
+
+Thank you for contributing to nf-core!
```

Comment:
I've taken to surrounding these kinds of advice statements in HTML comments, as people often don't delete them..
```suggestion
<!--
Please see uploading to`nf-core/configs` for more details:
https://github.com/nf-core/configs#uploading-to-nf-coreconfigs

Thank you for contributing to nf-core!
-->
```